### PR TITLE
update rj_gameplay/README.md

### PR DESCRIPTION
**Pull request by kfu02**
## Description
Updated to reflect new SRTP structure.

### CODE REVIEW

File: rj_gameplay/README.md [hunk 1]
```diff
@@ -11,32 +11,39 @@ Stable code for gameplay.
...
 
-Handles complex single robot behavior. Think of as one role in a play.
-(Examples: Goalie Receiver Waller)
+Coordinates one or more Roles at a high level. Generates Role Requests. (e.g. PassTactic, WallTactic)
 
 **Play - `rj_gameplay/play/`**
 
-Handles multi-robot behavior. Gets potential roles from a list of >=6 tactics,
-then assigns them as it sees fit. (Examples: Basic Defense)
+Coordinates all robots on the field for a given situation. Gets role requests from its tactics, then assigns them according to the cost functions of those role requests. (Examples: Basic122, Basic Defense)
```
Comment (kasohrab) [inline rj_gameplay/README.md]:
```suggestion
Coordinates all robots on the field for a given situation. Gets role requests from its tactics, then assigns them according to the cost functions of those role requests. (Examples: Basic Offense, Basic Defense)
```

File: rj_gameplay/README.md [hunk 2]
```diff
@@ -11,32 +11,39 @@ Stable code for gameplay.
...
+
+The general flow of how a RobotIntent (or specific command for the robot) is created is as follows:
+
+GameplayNode -> SituationAnalyzer -> Play (selected based on situation) -> Tactic(s) -> Role(s) -> Skill(s)
```
Comment (kasohrab) [inline rj_gameplay/README.md]:
```suggestion
GameplayNode -> Play Analysis/Selector -> Play (selected based on situation) -> Tactic(s) -> Role(s) -> Skill(s)
```

**Review by kasohrab — CHANGES_REQUESTED** [review]
these should be updated based on my latest pr

File: rj_gameplay/README.md [hunk 3]
```diff
@@ -11,32 +11,39 @@ Stable code for gameplay.
...
-(Examples: Goalie Receiver Waller)
+Coordinates one or more Roles at a high level. Generates Role Requests. (e.g. PassTactic, WallTactic)
 
 **Play - `rj_gameplay/play/`**
 
-Handles multi-robot behavior. Gets potential roles from a list of >=6 tactics,
-then assigns them as it sees fit. (Examples: Basic Defense)
+Coordinates all robots on the field for a given situation. Gets role requests from its tactics, then assigns them according to the cost functions of those role requests. (Examples: Basic122, Basic Defense)
+
+The general flow of how a RobotIntent (or specific command for the robot) is created is as follows:
```
Comment (parkerra) [inline rj_gameplay/README.md]:
```suggestion
The general flow of how a RobotIntent (or specific command for the robot) is created as follows:
```

File: rj_gameplay/README.md [hunk 4]
```diff
@@ -49,6 +56,8 @@ function will be replaced by ROS Actions soon.)
 Uses SituationAnalyzer to select the best play to run, calling tick() on the
```
Comment (kasohrab) [inline rj_gameplay/README.md]:
```suggestion
Uses basic_play_selector to select the best play to run, calling tick() on the
```

**Review by kasohrab — CHANGES_REQUESTED** [review]
and this

**Review by kasohrab — APPROVED** [review]
looks good to me

**Review by parkerra — APPROVED** [review]

**Pull request merged**